src/components/TextInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+function RenderTextField({ ...props }) {
+  const textProp = Object.assign({}, props);
+  delete textProp.meta;
+  delete textProp.input;
+
+  return (
+    <TextField
+      {...textProp}
+    />
+  );
+}
+const TextInput = ({ name, ...props }) => (
+  <Field
+    {...props}
+    name={name}
+    id={name}
```

Comment:
id is needed because its a property pass to TextField so in reality Name, id are the same thing but each library names them differently